buildscripts/Build.proj

```diff
@@ -207,24 +221,37 @@ limitations under the License.
 
 	</Target>
 
-
 	<Target
-		Name="_ExecStatLight"
+		Name="_ExecNUnit_Without_IntegrationTest"
```

Comment:
It would be great if we didn't have a copy of the `_ExecNUnit` target just to change the excluded categories, maybe just set a property for `AdditionalNUnitArguments` and call `_ExecNUnit`. However, I'm happy enough to merge this as is.